frontend/target/app/view/catalog/Tree.js

```diff
@@ -50,7 +50,13 @@ Ext.define('Target.view.catalog.Tree', {
                     text: 'Name',
                     flex: 2,
                     sortable: true,
-                    dataIndex: 'text'
+                    dataIndex: 'text',
+                    renderer: function (value, metadata, record) {
```

Comment:
Esse tb funciona mais é o jeito dificil, vc poderia simplesmente ter passado x-fa fa-bookmark nesse atributo https://github.com/linea-it/dri/pull/463/files#diff-1de15d425e01569b7707a42f80e63ae8L154

Outra coisa não use a classe fa nos icones do Fontawesome use x-fa é o parse que o Extjs tem para esses icones
ficaria 'x-fa fa-bookmark'